client/app/cdn/dedicated/manage/logs/cdn-dedicated-manage-logs.html

```diff
@@ -0,0 +1,32 @@
+<div data-ng-controller="CdnLogsCtrl as listCtrl">
+
+    <ovh-tail-logs>
+        <div class="row ml-0"
+             data-ng-repeat="log in listCtrl.logger.logs track by $index"
+             data-ng-class="{
+                'text-primary' : log.message.status >= 200 && log.message.status < 300,
+                'text-warning' : log.message.status >= 300 && log.message.status < 499,
+                'text-danger'  : log.message.status > 499
+            }">
+
+            <div class="col-md-12">
```

Comment:
No need for a
```js
<div class="row">
    <div class="col-md-12">
    </div>
</div>
```

A `div` is already as long as possible :)

You can write instead:
```js
<div
    class="ml-0"
    data-ng-repeat="...">
    <span ...></span>
</div>
```